Bottom: e384a139af51ff4413b8f9d350e0f184c961fed1
Top:    22b020f051838e7e16afe34eea476e75a3ba81c5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 01:18:07 +0100

DEBUG DEBUG DEBUG DEBUG


---

diff --git a/src/draw_funcs.c b/src/draw_funcs.c
index 363a85e..935fb74 100644
--- a/src/draw_funcs.c
+++ b/src/draw_funcs.c
@@ -379,8 +379,8 @@ draw_layer (LayerType *layer, const BoxType *drawn_area, void *userdata)
 
   /* draw vias */
   r_search (PCB->Data->via_tree, drawn_area, NULL, via_inlayer_callback, layer);
-  r_search (PCB->Data->pin_tree, drawn_area, NULL, hole_callback, NULL);
-  r_search (PCB->Data->via_tree, drawn_area, NULL, hole_callback, NULL);
+//  r_search (PCB->Data->pin_tree, drawn_area, NULL, hole_callback, NULL);
+//  r_search (PCB->Data->via_tree, drawn_area, NULL, hole_callback, NULL);
 }
 
 struct draw_funcs d_f = {
